assets/js/modules/search-console/components/dashboard/DashboardPopularKeywordsWidget.js

```diff
@@ -62,15 +62,57 @@ function DashboardPopularKeywordsWidget( { Widget, WidgetReportZero, WidgetRepor
 	} ) );
 
 	if ( loading ) {
-		return <PreviewTable padding />;
+		return (
+			<Widget
+				noPadding
+				Footer={ () => (
+					<SourceLink
+						className="googlesitekit-data-block__source"
+						name={ _x( 'Search Console', 'Service name', 'google-site-kit' ) }
+						href={ baseServiceURL }
+						external
+					/>
+				) }
```

Comment:
Same here, extract it to an anonymous component and use it for all footers instead of duplicating the code.